runtimes/pythonrt/runner/values.py

```diff
@@ -0,0 +1,127 @@
+"""AutoKitteh values
+
+Wraps and unwraps autokitteh values.
+"""
+
+import pb.autokitteh.values.v1.values_pb2 as pb
+from collections import namedtuple
+from datetime import datetime, timedelta, UTC
+
+from google.protobuf.duration_pb2 import Duration
+from google.protobuf.timestamp_pb2 import Timestamp
+
+
+class ValueWrapper:
```

Comment:
IMO no need for a class, only two functions in this module.
In general, if you have a method that does  not use `self` at all, reconsider.

Or if you really want the class, make this a staticmethod and then it's accessible from the class level as well.

Reply:
got rid of the class. i'll add it back if needed in the future.